app/views/list/list.erb

```diff
@@ -59,7 +59,7 @@
           <td class="main-link" itemprop='itemListElement' itemscope itemtype='http://schema.org/ListItem'>
             <meta itemprop='position' content='<%= i + 1 %>'>
             <span class="link-top-line">
-              <a itemprop='url' href='<%= t.url %>' class='title raw-link raw-topic-link'><%= t.title %></a>
+              <a itemprop='url' href='<%= t.url %>' class='title raw-link raw-topic-link'><%= t.fancy_title %></a>
```

Comment:
Does this change cause issues with quotes, apostrophes and the like? When I make this change locally I see this: 

<img width="2262" height="186" alt="Image" src="https://github.com/user-attachments/assets/74717118-d7a2-4976-bbb2-a375ad09ce2f" />